vcd_cli/network.py

```diff
@@ -512,3 +518,37 @@ def delete_external_network(ctx, name):
             stdout('External network deleted successfully.', ctx)
         except Exception as e:
             stderr(e, ctx)
+
+
+@external.command('update', short_help='update an external network')
```

Comment:
'update name and description of an external network' since scope is only to update name and description